contracts/external/rari/IRewardsDistributor.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.8.0;
+
+interface IRewardsDistributor {
```

Comment:
Call this IRewardsDistributorAdmin and move it a new top-level folder called `feirari` with the rest of the contracts in this PR